tests/integration/targets/redis_info/meta/main.yml

```diff
@@ -0,0 +1,2 @@
+dependencies:
+- setup_redis_replication
```

Comment:
Shouldn't that role be called `setup_redis`? Or are you specifically setting up redis with multiple instances and some sort of replication? (If yes, why?)

Reply:
it would be cool to have such stuff preconfigured in CI, imo.
as i can see, there's no module for replication setup there but it's a commond task.
When somebody decide to implement that, this will make his/her life easier.
@levonet , if you decide to create the redis_replication module, please let me know before starting.

Reply:
I don't understand, is it good or bad?
`setup_redis_replication` lanch 2 redis services, one as master, another as slave.
`redis_info` module connects to both and gets different responses.
You or someone else will have to write tests on redys.py module. `setup_redis_replication` will allow you to test commands such as:
```yaml
- name: Activate slave mode
  redis:
    command: slave

- name: Deactivate slave mode
  redis:
    command: slave
    slave_mode: master
```

Reply:
I do not feel the need to change anything here.